src/main/resources/config/liquibase/changelog/20211118061600_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="akesfeden" id="20211118061600">
+        <addColumn tableName="exercise">
+            <column name="sample_solution_publication_date" type="datetime"/>
```

Comment:
![image](https://user-images.githubusercontent.com/65814168/144748140-2cc153d7-7953-4434-a361-fb21708b5b5d.png)
https://docs.artemis.ase.in.tum.de/dev/guidelines/server/#database
I just finished my own PR where I add a similar change and noticed in our documentation that we should avoid using datetime but use timestamp instead. I remember looking at your PR and wanted to check and tell you about this 😅

Reply:
Timestamp was also my initial attempt according to the documentation you mentioned, until @krusche asked me to change the database type to `datetime` and I did so at https://github.com/ls1intum/Artemis/pull/4328/commits/22fb965763e38a3c6578e59a2a9f9442e615cb64.

I am not sure about the reasoning behind, maybe we can talk about it and extend documentation if there can be cases like this.

Reply:
Oh ok, I think this might be important to bring up in the next developer meeting so that everyone is up to date what we should use 😅

Reply:
Relevant documentation is updated at https://github.com/ls1intum/Artemis/pull/4486

Reply:
```suggestion
            <column name="example_solution_publication_date" type="datetime"/>
```

Reply:
Ideally, also rename those from `sample` to `example` within the same liquibase migration
<img width="412" alt="image" src="https://user-images.githubusercontent.com/744067/146277990-5832ac5b-712c-49ca-889d-c4ffd5701554.png">

Reply:
Thanks @krusche, I'll rename it. However, we already deployed it to TS3, so would renaming it in the same Liquibase file cause inconsistencies? If yes, I can add a new migration to rename the same column. Which one do you suggest?

Reply:
`sample_solution` (and similar) fields renamed to `example_solution` (and similar).